pkg/webhook/deploy/deployment.go

```diff
@@ -46,6 +53,51 @@ func Webhook(ctx context.Context, cl runtimeclient.Client, s *runtime.Scheme, na
 	return nil
 }
 
+// Delete deletes the webhook app if it's deployed. Does nothing if it's not.
+// Returns true if the app was deleted.
+func Delete(ctx context.Context, cl runtimeclient.Client, s *runtime.Scheme, namespace string, oldObjectOnly bool) (bool, error) {
+	objs, err := getTemplateObjects(s, namespace, "0", []byte{00000001})
```

Comment:
it's probably just a detail, I know that it's not being applied anywhere - I wouldn't use zero `0` as a value for the image url just to be sure that is not expressed as a number while the image location should be a string
```suggestion
	objs, err := getTemplateObjects(s, namespace, "dummy-image", []byte{00000001})
```
apart from that, this looks more natural than the zero :-)